18F/identity-idp
# LG-8336: Implement post timer actions

## 🎫 Ticket

[LG-8336](https://cm-jira.usa.gov/browse/LG-8337)

## 🛠 Summary of changes

This ticket completes the implementation of the the timer that counts down to when the one-time password expires. Now, we redirect to a page that informs users that their code has expired and to request a new one.

## 📜 Testing Plan

Provide a checklist of steps to confirm the changes.

Before testing
- add `otp_valid_for: 3` to `application.yml` to speed up testing

1. Spin up application and log on to `http://localhost:3000
2. Create new account and follow steps for account creation
   1. On account creation, select phone and backup codes as MFA options for faster testing
3. Enter phone number to add to the account
4. On the one time password screen, wait approximately 30 seconds for the countdown banner to appear. The countdown will count down from the set `otp_valid_for` value
5. Wait until the timer reaches zero
6. Redirect to OTP expiration page

Optional: Repeat testing on add phone setup and one time code verification setup on existing accounts.

## 👀 Screenshots

If relevant, include a screenshot or screen capture of the changes.
![Screenshot of one-time code expired page](https://user-images.githubusercontent.com/17969963/213775195-ae9975ba-b7ca-477f-bffe-d055b5fcff75.png)
